internal/pkg/service/buffer/worker/service/slice_upload_test.go

```diff
@@ -171,50 +171,46 @@ func TestSliceUploadTask(t *testing.T) {
 
 	// Check "close slice" logs
 	wildcards.Assert(t, `
-[orchestrator][slice.close]INFO  ready
-[orchestrator][slice.close]INFO  assigned "00000123/my-receiver-1/my-export-1/0001-01-01T00:00:01.000Z/0001-01-01T00:00:01.000Z"
 [task][slice.close/%s]INFO  started task "00000123/my-receiver-1/my-export-1/slice.close/%s"
 [task][slice.close/%s]DEBUG  lock acquired "runtime/lock/task/00000123/my-receiver-1/my-export-1/slice.close/%s"
 [task][slice.close/%s]INFO  waiting until all API nodes switch to a revision >= %s
```

Comment:
Upraveny nestabilny test.

Kazdy velky check som rozdelil na 3 samostatne.

Ako sa tu pouzivaju `mocked clock`, tak virtualne je rozdel v sekundach, ale realne to bezi hned za sebou.

Tj. log hlasky sa mozu pomiesat.

Mozno by mal `DebugLogger` vzdy volat `Sync` po `Write` :man_shrugging:  (v produkcii to nechces, spomalovalo by to apku).